contents/docs/integrate/_snippets/install-flutter.mdx

```diff
@@ -17,13 +17,13 @@ Remember that the application lifecycle events won't have any special context se
         <meta-data android:name="com.posthog.posthog.API_KEY" android:value="<ph_project_api_key>" />
         <!-- usually 'https://app.posthog.com' or 'https://eu.posthog.com' -->
         <meta-data android:name="com.posthog.posthog.POSTHOG_HOST" android:value="<ph_instance_address>" />
-        <meta-data android:name="com.posthog.posthog.TRACK_APPLICATION_LIFECYCLE_EVENTS" android:value="false" />
-        <meta-data android:name="com.posthog.posthog.DEBUG" android:value="false" />
+        <meta-data android:name="com.posthog.posthog.TRACK_APPLICATION_LIFECYCLE_EVENTS" android:value="true" />
```

Comment:
does this mean flutter SDK now has autocapture?

Reply:
already had https://posthog.com/docs/libraries/flutter#autocapture
but not click events and some other stuff as JS, but it does capture a few stuff automatically

Reply:
Cool! Can we update the top of the docs to reflect this?

<img width="728" alt="Screenshot 2024-02-13 at 10 33 31 AM" src="https://github.com/PostHog/posthog.com/assets/7090054/3b59a53f-44bb-49a9-bb0e-2ab5cd7003bd">

Reply:
discussed offline